share/completions/figlet.fish

```diff
@@ -0,0 +1,29 @@
+function __fish_print_figlet_fonts
+    find (figlet -I2) -type f ^ /dev/null | sed -e 's$/.*/\([^/.]*\)\.[ft]lf$\1\tFont$;tx;d;:x'
```

Comment:
You're solution is much faster (yours: 4ms; sed: 12ms; find + match: 14ms). I'll commit it.